src/hypothesis/extra/django/models.py

```diff
@@ -115,7 +115,7 @@ def __init__(self, model, mappings):
             strategy=st.fixed_dictionaries(mappings))
 
     def __repr__(self):
-        return u'ModelStrategy(%s)' % (self.model.__name__,)
+        return 'ModelStrategy(%s)' % (self.model.__name__,)
```

Comment:
Note that this will still fail if __name__ is non-ascii because \_\_repr\_\_ isn't actually allowed to return non-ascii strings in Python 2.